### PR TITLE
Doc: shape management in tyr

### PR DESCRIPTION
**Pull request by pbougue**
TODO:
- [x] Check that PUT is working that way

### CODE REVIEW

File: source/tyr/readme.md [hunk 1]
```diff
@@ -363,19 +373,94 @@ Get all a user information:
 }
 ```
 
+#### GET Parameters
+
+name             | description                                                                          | required | default        |
+-----------------|--------------------------------------------------------------------------------------|----------|----------------|
+disable_geojson  | if true the shape will be fully displayed (otherwise displaying: `null` or `{}`)     | nope     | false          |
```
Comment (datanel) [inline source/tyr/readme.md]: disable_geojson=true by default

File: source/tyr/readme.md [hunk 2]
```diff
@@ -363,19 +373,94 @@ Get all a user information:
...
+            },
+            "properties": {
+                "prop0": "value0",
+                "prop1": {
+                    "this": "that"
+                }
+            }
+        },
+        "has_shape": true,
+        "shape": {},
```
Comment (datanel) [inline source/tyr/readme.md]: you can remove these 2 lines above

File: source/tyr/readme.md (hunk 1 above)
Comment (pbougue) [reply source/tyr/readme.md]: done

File: source/tyr/readme.md (hunk 2 above)
Comment (pbougue) [reply source/tyr/readme.md]: done

**Comment by datanel** [comment]
ok for me

**Review by datanel — APPROVED** [review]

**Pull request merged**